docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+to deploy an AWS ECS service.
+
+Once the integration role permissions are configured, the wizard walks you
+through deploying the Teleport database service in a high availability (HA)
+setup on AWS Fargate for AWS ECS.
+
+First, it creates an IAM role for Teleport database service instances to use.
+This role has permissions to output logs in ECS, describe RDS databases, and
+connect to RDS databases.
+
+Next, it creates an ECS task definition that includes various settings for
+configuring and starting a Teleport database service instance. 
+The task definition uses the IAM role created in the first step as both the
+[ECS task execution role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html).
+and the
+[ECS task role](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html)
+
+The wizard then creates an ECS deployment in the same AWS VPC as your RDS
+databases with two copies of the Teleport database service instance task for
+an HA setup.
+By deploying the Teleport database service instances in the same VPC as the RDS
+databases, the instances can connect to databases in private subnets.
+
+Once an ECS task starts, the Teleport database service instance establishes
+a tunnel connection to your Teleport cluster.
+
+This is a serverless HA approach to deploying the Teleport database service.
+It does not require any allowed inbound traffic from the internet and supports
+[automatic agent upgrades](../../../upgrading/automatic-agent-updates.mdx).
+
+The Teleport database service instances use
+[dynamic database registration](../../../enroll-resources/database-access/guides/dynamic-registration.mdx)
+so that you can enroll or disenroll databases without restarting or modifying
+the deployment.
+The instances will match and proxy connections for all dynamic `db` resources in
+Teleport that have the following labels:
+
+```yaml
+account-id: "<AWS account ID>"
+region: "<AWS region>"
+vpc-id: "<AWS VPC ID>"
+```
+
+{/* TODO(gavin): add the deployment diagram here once it is finalized */}
+
+AWS resources created by the RDS enrollment wizard are tagged in AWS so that you
+can search and export them using the
+[AWS Resource Groups / Tag Editor](https://console.aws.amazon.com/resource-groups/tag-editor/find-resources).
+The following tags are applied:
+
+```code
+teleport.dev/cluster      <Var name="cluster-name"/>
+teleport.dev/origin       integration_awsoidc
+teleport.dev/integration  <Var name="my-integration"/>
+```
+
+You can also search for AWS resources created by the wizard using the `aws` cli:
+
+```code
+$ aws resourcegroupstaggingapi get-resources \
+--region <Var name="us-west-1"/> \
+--tag-filters \
+Key=teleport.dev/cluster,Values=<Var name="cluster-name"/> \
+Key=teleport.dev/integration,Values=<Var name="my-integration"/> \
+Key=teleport.dev/origin,Values=integration_awsoidc
+```
+
+## How to find the ECS deployment
```

Comment:
I would put this section under "Troubleshooting" so a user who wants to navigate right to the Troubleshooting section can see it quickly.

Reply:
moved in 2b316271e5fd9496db4c106da93de4e45005f1c6